### PR TITLE
MWPW-136317 Dropzone enabled on L1 verbs (plus compress) except rotate

### PR DESCRIPTION
**Pull request by puruadobe**
Dropzone enabled for the following L1 verbs plus compress (exclude rotate), rest of the verbs will have it only for the first visit.

Resolves https://jira.corp.adobe.com/browse/MWPW-136317

Before:
https://stage--dc--adobecom.hlx.live/acrobat/online/sign-pdf?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/request-signature?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/pdf-editor?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/delete-pdf-pages?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/compress-pdf?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/rearrange-pdf?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/split-pdf?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/add-pages-to-pdf?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/extract-pdf-pages?martech=off
https://stage--dc--adobecom.hlx.live/acrobat/online/crop-pdf?martech=off 

After:
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/sign-pdf?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/request-signature?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/pdf-editor?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/delete-pdf-pages?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/compress-pdf?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/rearrange-pdf?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/split-pdf?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/add-pages-to-pdf?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/extract-pdf-pages?martech=off
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/crop-pdf?martech=off 
https://puru-mwpw-136317--dc--puruadobe.hlx.live/acrobat/online/add-pdf-page-numbers?martech=off

### CODE REVIEW

File: acrobat/blocks/dc-converter-widget/dc-converter-widget.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ export default async function init(element) {
     })
   }
 
-  if (VERB === 'compress-pdf' || VERB === 'reorder-pages' || preRenderDropZone) {
+  if (window.browser?.isMobile && (VERB === 'fillsign' || VERB === 'sendforsignature' || VERB === 'add-comment' || VERB === 'crop-pages' || VERB === 'number-pages')) {
```
Comment (peller) [inline acrobat/blocks/dc-converter-widget/dc-converter-widget.js:321]: need to leave the existing conditions as well
See comment on ticket, we may be able to extend this further

**Review by peller — CHANGES_REQUESTED** [review]
must update conditions before merging

File: acrobat/blocks/dc-converter-widget/dc-converter-widget.js (hunk 1 above)
Comment (puruadobe) [reply acrobat/blocks/dc-converter-widget/dc-converter-widget.js:321]: @peller working on it as per the request in the ticket.

**Review by peller — APPROVED** [review]

**Comment by puruadobe** [comment]
Formatted only my code changes so that it's easy to review.

File: acrobat/blocks/dc-converter-widget/dc-converter-widget.js (hunk 1 above)
Comment (puruadobe) [reply acrobat/blocks/dc-converter-widget/dc-converter-widget.js]: Done

**Review by Blainegunn — APPROVED** [review]
looks good, thank you @puruadobe

**Comment by peller** [comment]
@puruadobe @Blainegunn is there anything blocking here? Can we get this on stage?

**Pull request merged**